sources/__tests__/getRole.js

```diff
@@ -89,6 +89,7 @@ const cases = [
 	["html", null, createElementFactory("html", {})],
 	["iframe", null, createElementFactory("iframe", {})],
 	["img with alt=\"some text\"", "img", createElementFactory("img", {alt: "text"})],
+	["img with missing alt", "img", createElementFactory("img", {})],
```

Comment:
per discussion https://github.com/eps1lon/dom-accessibility-api/issues/429#issuecomment-696259998 I think we agree on this behavior @eps1lon?

Reply:
Perfect, thanks!